cl/disclosures/tasks.py

```diff
@@ -476,3 +523,27 @@ def import_disclosure(self, data: Dict[str, Union[str, int, list]]) -> None:
 
     # Save PDF content
     save_disclosure(extracted_data=content, disclosure=disclosure)
+    # Remove disclosure ID in redis for completed disclosure
+    interface.delete(disclosure_key)
```

Comment:
Do we need to delete this key on line 522 as well?

Reply:
Yes